Source/Core/Core/AchievementManager.cpp

```diff
@@ -729,6 +734,12 @@ AchievementManager::RichPresence AchievementManager::GetRichPresence()
   return rich_presence;
 }
 
+const std::map<std::string, std::unique_ptr<OSD::Icon>>*
+AchievementManager::GetChallengeIcons() const
```

Comment:
I would return a reference here instead of a pointer (unless you end up copying m_active_challenges like I mention in my comment about locking, in which case just return by value).